### DISCUSSION TITLE
Feature request - submit this software to winget package manager

### DISCUSSION THREAD

**Discussion by mjkl-gh** [snappysnappydog/Simple-IP-Config#113] [Ideas]
Microsoft released a package manager for windows called winget. I'm currently consolidating my developer config around installing most programs through winget. As I use your program almost daily, it would be really convenient to have it featured on winget as well. A lot of other (open source) projects are featured there as well 

Documentation on how to be pulled into the package manager is at:
https://docs.microsoft.com/en-us/windows/package-manager/package/repository

I hope you will consider it!

**Comment by ali50m** [comment]
+1

**Comment by KurtisLiggett** [comment]
I had never heard of winget. I will look into it.
